python/plugins/processing/gui/wrappers.py

```diff
@@ -240,6 +259,11 @@ def value(self):
 
 class BooleanWidgetWrapper(WidgetWrapper):
 
+    def __init__(self, param, dialog, row=0, col=0, **kwargs):
```

Comment:
Instead of this, could you override createLabel() with something like:

    def createLabel(self):
        if self.dialogType == DIALOG_STANDARD:
            return None
        else:
            return super().createLabel()